Dockerfile.downstream

```
@@ -1,27 +1,31 @@
+ARG TARGETARCH
+ARG COMMIT
+
 # Build the manager binary
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23 as builder
+FROM --platform=linux/$TARGETARCH brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23 as builder
 
+ARG TARGETARCH=amd64
 ARG VERSION="unknown"
 ARG COMMIT
+ARG LDFLAGS
 
 WORKDIR /opt/app-root
 
 # Copy the go manifests and source
-COPY .git/ .git/
 COPY bpf/ bpf/
 COPY cmd/ cmd/
 COPY pkg/ pkg/
 COPY vendor/ vendor/
 COPY go.mod go.mod
 COPY go.sum go.sum
-COPY Makefile Makefile
-COPY .mk/ .mk/
 
 # Build
-RUN make compile
+RUN GOARCH=$TARGETARCH go build -ldflags "$LDFLAGS" -mod vendor -a -o bin/netobserv-ebpf-agent cmd/netobserv-ebpf-agent.go
```

If you don't mind to fix FIPS at the same time :-)

```suggestion
ENV GOEXPERIMENT strictfipsruntime
RUN GOARCH=$TARGETARCH go build -tags strictfipsruntime -ldflags "$LDFLAGS" -mod vendor -a -o bin/netobserv-ebpf-agent cmd/netobserv-ebpf-agent.go
```